Designs/create_shoppinglist.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Create BucketList</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,300,100" rel="stylesheet">
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+
+<form class="card" id="new_bkt">
+    <div class="form-header">
+        <h4 class="title">Create A new Shopping list</h4>
```

Comment:
To make your app feel more professional, I'd suggest making sure your text/sentences are well punctuated. So for instance rather than `Create A new Shopping list` you could have `Create a new shopping list`. Note the difference in capitalization of first letters. The latter reads more like actual english.

You should try and change this throughout your app, since it's an easy change that makes a lot of difference, I think.

Reply:
Noted.